docs/natspec-format.rst

```diff
@@ -26,7 +26,9 @@ Documentation Example
 =====================
 
 Documentation is inserted above each ``class``, ``interface`` and
-``function`` using the doxygen notation format.
+``function`` using the doxygen notation format. Note: a ``public``
```

Comment:
Perhaps start the sentence beginning with `Note: ` on a new line?

Reply:
fixed in 99086d07bc4632d8e3f922f4e28db8b6cb08dbfa